mlflow/server/js/src/common/utils/ActionUtils.js

```diff
@@ -37,6 +39,12 @@ export const rejected = (apiActionType) => {
  * response from the server. Defaults to 1 second.
  */
 export const wrapDeferred = (deferred, data, timeLeftMs = 60000, sleepMs = 1000) => {
```

Comment:
FAR: You should look at the X-JWT-Token header in each request to refresh the token if it is present. I know that this feature is not included in the python lib yet.